Modules/_decimal/_decimal.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008-2020 Stefan Krah. All rights reserved.
+ * Copyright (c) 2008-2012 Stefan Krah. All rights reserved.
```

Comment:
This change seems odd; is this definitely correct? (Should "2012" have been "2021" here?)

I don't think it's terribly consequential, and I'm happy for the change to stay if that keeps us compatible with upstream.